### DISCUSSION TITLE
Tab with [<block colored>] and wording Focus in title

### DISCUSSION THREAD

**Discussion by AddictArts** [zellij-org/zellij#3335] [Q&A] [answered]
What is it?

It happens once in awhile and I don't know what triggered it or what it is. In the menu's there is no focus key sequence that I can find. How do you exit that "focus" thing?

**Comment by giyany** [comment]
It's difficult for me to understand what's happening from your description. Can you explain what you see, or attach a screenshot?

Is it possible that you turned on the scrollback search? That'll be Ctrl s + s, then you enter a term to search for. the term will be highlighted in color.

**Comment by AddictArts** [comment]
@giyany Thanks for asking.  Not the scrollback though.

The tabs at the top normally appear as `>Tab #1>Tab #2>` and so on. However, once in awhile I do something, key combo? that changes them to say `>Tab Focus [ ]>Tab #2`

I can't do a screenshot, because I don't know how to enter or exit this state.  I can't find any docs on "Focus" and in the `[ ]` is a solid square of a certain color like pink.

**Reply by Lockszmith-GH** [reply, marked as answer]
the `[ ]` appears when there are multiple connections viewing the tab

![image](https://github.com/zellij-org/zellij/assets/905716/effcdd65-6d2a-479e-9f79-f207299d0015)

As you can see, both windows are connected to the same zellij session. (You can also see that the sessions size constraint each other - the short window forces the hight, the narrow window forces the width of the session)

But I'm not sure about `Focus`

**Reply by AddictArts** [reply]
@Lockszmith-GH Thanks. Ok I don't believe I had two open, but must have. Trying not to ask the obvious question, but I guess you can close the other session? kill it from a terminal. The BLOCK is very distracting and unnerving and if another has it open is cause for concern, things are going to go very wrong if you continue.

**Reply by AddictArts** [reply]
Here is where the Focus is

![Screenshot 2024-05-09 132033](https://github.com/zellij-org/zellij/assets/911906/f5d94410-48ae-4acb-a1ed-178bba9eaea4)

**Reply by AddictArts** [reply]
Ok I found it `Ctrl o` and the `w` for the session manager. Then use the >Attach to Session> to disconnect others with `Ctrl x`.
